protocol/0038-OLIQ-liquidity_provision_order_type.md

```diff
@@ -60,10 +60,20 @@ Input data:
 
 Steps:
 
+1. From the market parameter - to be set as part of [market proposal](0028-GOVE-governance.md)  `market.liquidity.volumeRange` which is a percentage price move (e.g. `0.05 = 5%` and from `mid_price` calculate:
```

Comment:
I think this implies a governance spec change too?

Reply:
I checked; the governance spec for market changes only lists the immutable fields, but not all the mutable ones, so we're fine.